pubsub/awssnssqs/awssnssqs.go

```diff
@@ -186,9 +186,9 @@ func (o *URLOpener) OpenTopicURL(ctx context.Context, u *url.URL) (*pubsub.Topic
 	if err != nil {
 		return nil, fmt.Errorf("open topic %v: %v", u, err)
 	}
-	client := sns.New(o.ConfigProvider, overrideCfg)
```

Comment:
Please also remove the `sns.New` and `sqs.New` from `var Set` above, near line 121.